# A stamped 2D wind vector

Header header
float64 speed
float64 direction